### DISCUSSION TITLE
Remap opening links

### DISCUSSION THREAD

**Discussion by wiljam144** [nvim-neorg/neorg#164] [Q&A] [answered]
The default for opening links is `<CR>` is there a way to remap it?

**Comment by mrossinek** [comment, marked as answer]
Although the "megalist" of keybindings in the wiki is a little bit outdated, the instructions on how to change default keybindings is still valid: https://github.com/nvim-neorg/neorg/wiki/User-Keybinds

For the specific example of remapping the `goto_link` function, see [e.g. my config](https://gitlab.com/mrossinek/dotfiles/-/blob/master/nvim/.config/nvim/after/plugin/neorg.lua#L79)

**Reply by wiljam144** [reply]
thank you
